addons/rookupgrade/10to14/install.sh

```diff
@@ -91,6 +97,11 @@ function rookupgrade_10to14_upgrade() {
         kubectl -n rook-ceph set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v1.1.9
         kubectl -n rook-ceph set image deploy/rook-ceph-tools rook-ceph-tools=rook/ceph:v1.1.9
 
+        log "Awaiting up to 5 minutes to check Rook Ceph Pod(s) are Running"
+        if ! spinner_until 300 check_for_running_pods "rook-ceph"; then
```

Comment:
I don't understand what check_for_running_pods does that `"$DIR"/bin/kurl rook wait-for-rook-version` does not do. If there is some gap in functionality is it possible to add that to the wait-for-rook-version command rather than add more bash?